man/futile.logger-package.Rd

```diff
@@ -85,7 +85,8 @@
   appender generator functions:
 
   \describe{ \item{appender.file}{Write to a file}
-  \item{appender.console}{Write to the console} }
+  \item{appender.console}{Write to the console}
+  \item(appender.tee}{Write to both a file and the console} }
```

Comment:
Looks like you have a syntax error here that is breaking the build. Notice the '(' instead of '{'. See [the build log](https://travis-ci.org/zatonovo/futile.logger/builds/47367169) for details.